code/modules/projectiles/gun.dm

```diff
@@ -56,6 +56,8 @@
 	var/knife_x_offset = 0
 	var/knife_y_offset = 0
 
+	var/can_holster = TRUE // Can it be holstered?
```

Comment:
Can you make this in the autodoc format?
```
/// comment here
var/variable
```

Reply:
Don't do it. This doesn't need documentation.

Also remove that comment, as it does not improve the readability. The name clearly conveys what this variable is for.